src/app/cronjobs.rs

```diff
@@ -1,4 +1,4 @@
-use k8s_openapi::{api::batch::v1beta1::CronJob, chrono::Utc};
+use k8s_openapi::{api::batch::v1::CronJob, chrono::Utc};
```

Comment:
note that [CronJob is stable since 1.21](https://kubernetes.io/blog/2021/04/09/kubernetes-release-1.21-cronjob-ga/)

Reply:
Thanks for the catch